chrome/content/zotero/bindings/styled-textbox.xml

```diff
@@ -339,14 +346,13 @@
 				}
 				
 				this.htmlToRTF = function(txt) {
-					// Catch this one before &nbsp; is clobbered by unescape
-					txt = txt.replace(/<span class=\"tab\">&nbsp;<\/span>/g, "\\tab{}");
-					txt = Zotero.Utilities.unescapeHTML(txt);
+					txt = this.convert("htmlRTF", txt);
 					for (var i=0,ilen=this._htmlRTFmap.length; i < ilen; i++) {
 						var entry = this._htmlRTFmap[i];
 						txt = txt.replace(entry[0], entry[1]);
 					}
-					txt = this.convert("htmlRTF", txt);
+					txt = Zotero.Utilities.unescapeHTML(txt);
+					txt = txt.replace(/[\x7F-\uFFFF]/g, function(aChar) { return "\\uc0\\u"+aChar.charCodeAt(0).toString()+"{}"});
```

Comment:
The editor sets high-bit characters (like endash) as HTML named entities, which can't be converted by a generic function. A side effect of `unescapeHTML` is to convert them to character strings, so that they can be RTF-escaped.